src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -112,6 +112,10 @@
 public class ForgeEventFactory
 {
 
+    /**
+     * @deprecated Use {@link #onPlayerMultiBlockPlace(EntityPlayer, List, EnumFacing, EnumHand)} instead.
+     */
+    @Deprecated
     public static MultiPlaceEvent onPlayerMultiBlockPlace(EntityPlayer player, List<BlockSnapshot> blockSnapshots, EnumFacing direction)
     {
```

Comment:
Dont duplicate code, bounce to the new one with the argument null, this makes it more obvious what to remove when deleting deprecated functions. And illustrates the parameter CAN be null.